spec/features/user_can_sign_out_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+class PostsTests < ActionDispatch::IntegrationTest
```

Comment:
We ended up not using Devise's helpers, so this class declaration is extraneous. Removed it.